tests/integration/components/power-calendar-multiple/days-test.js

```diff
@@ -78,5 +78,24 @@ module(
 
       assert.notOk(find('.ember-power-calendar-day[data-date="2013-10-06"]').disabled);
     });
+
+    test('If it receives `showDaysAround=false` option, it doesn\'t show the days before or after the first day of the month', async function(assert) {
+      assert.expect(3);
```

Comment:
Description is a little bit off -- before first day or after last day

Reply:
Updated to `before first or after last day of the month`